testing/trino-tests/src/test/resources/sql/trino/tpcds/iceberg/parquet/partitioned/q02.plan.txt

```diff
@@ -0,0 +1,51 @@
+remote exchange (GATHER, SINGLE, [])
+    local exchange (GATHER, UNKNOWN, [])
+        remote exchange (REPARTITION, ROUND_ROBIN, [])
+            join (INNER, PARTITIONED):
+                join (INNER, PARTITIONED):
+                    final aggregation over (d_week_seq)
+                        local exchange (GATHER, SINGLE, [])
+                            remote exchange (REPARTITION, HASH, [d_week_seq])
+                                partial aggregation over (d_week_seq)
+                                    final aggregation over (d_day_name, d_week_seq)
+                                        local exchange (GATHER, SINGLE, [])
+                                            remote exchange (REPARTITION, HASH, [d_day_name, d_week_seq])
+                                                partial aggregation over (d_day_name, d_week_seq)
+                                                    join (INNER, REPLICATED):
+                                                        local exchange (REPARTITION, ROUND_ROBIN, [])
+                                                            dynamic filter (ws_sold_date_sk::bigint)
+                                                                scan web_sales
+                                                            dynamic filter (cs_sold_date_sk::bigint)
+                                                                scan catalog_sales
+                                                        local exchange (GATHER, SINGLE, [])
+                                                            remote exchange (REPLICATE, BROADCAST, [])
+                                                                dynamic filter (d_week_seq::integer, d_week_seq::integer)
+                                                                    scan date_dim
+                    local exchange (GATHER, SINGLE, [])
+                        remote exchange (REPARTITION, HASH, [d_week_seq_20])
```

Comment:
Why were these numbers seemingly random? Can we make them deterministic? In my previous PR, these numbers seemed to change randomly in the plan, even though the output plan was the same.

Reply:
> Why were these numbers seemingly random? Can we make them deterministic?

These numbers are deterministic. If they are not, then it would suggest a (harmless) bug in planner

Reply:
Yup, usually the symbol names for a given query are stable across multiple runs.
If they aren't stable then any BaseCostBasedPlanTest tests would be constantly failing.